go/vm/lfvm/instructions.go

```diff
@@ -764,6 +772,18 @@ func opCreate(c *context) {
 		return
 	}
 
+	if c.revision >= vm.R12_Shanghai {
```

Comment:
maybe you could factor out the check into a helper function and then just call the check in the 2 ops, instead of having the same check twice ?